### DISCUSSION TITLE
When can we expect the release for 8.0

### DISCUSSION THREAD

**Discussion by shashankdevaraj** [fbaligand/kibana-enhanced-table#259] [General]
We need to upgrade the ELK, I am thinking to upgrade directly to 8

**Comment by fbaligand** [comment]
Well, I can’t give you a date.
I can only say that it’s on the roadmap.
You can follow the issue for that:
https://github.com/fbaligand/kibana-enhanced-table/issues/257

That said, I advise you (as Elastic do) to upgrade first to Kibana 7.17 first and then launch “assistant for Kibana 8.0 migration”.

**Reply by shashankdevaraj** [reply]
Thank you @fbaligand
